tools/infrasys/src/root.rs

```diff
@@ -40,6 +41,17 @@ pub fn check_root(root_role_path: &Path) -> Result<()> {
     Ok(())
 }
 
+async fn async_get_region() -> Result<String> {
+    let default_region_fallback = "us-east-1";
```

Comment:
I think I'd prefer to codify this as a constant somewhere so everything uses the same default region.  Nothing in the SDK provides anything like this?

Reply:
The SDK has something called `RegionProviderChain` which we use in **testsys** to set the default to `us-west-2`. We could use that same for **infrasys**.